CMakeLists.txt

```diff
@@ -374,3 +375,19 @@ foreach(i ${BENCH_FILES})
 	add_executable(bench_${name} ${i})
 	target_link_libraries(bench_${name} lcf)
 endforeach()
+
+# tools
+if(NOT DISABLE_TOOLS)
+	set(TOOLS lcf2xml)
```

Comment:
I think it would be cleaner to do it the other way around:
Empty tool list, add lcf2xml when not disabled.